### DISCUSSION TITLE
This is an amazing project

### DISCUSSION THREAD

**Discussion by jdluzen** [saddam213/OnnxStack#11] [General]
I just want to say thank you for this amazing project. It fills a desperate need in the C# community.

**Comment by saddam213** [comment]
Thanks for the kind words :)

Yeah, I struggled a lot trying to get into AI due to everything mainly being python only.

Hoping this will be a .NET version of diffusers if possible, thats a huge task, but bit by bit Ill try get there, would love any contributions you can think to add :)

I dont know python at all so that can take me a bit longer to include new things, as most things need to be written from scratch, so if anyone knows python well let me know, I have many sections I would like to be reviewed :p

**Reply by jdluzen** [reply]
Same. It was, and still is, a huge struggle due to Python. I'm using an AMD GPU, so that's even more of a struggle on top of that.

I have some ideas for a contribution, and now that I can actually tinker with code, I can see if it'll actually work.

Written from scratch? Could have fooled me! Code looks super clean.

**Comment by Amin456789** [comment]
agreed! i love it so much

**Comment by Amin456789** [comment]
i'm gonna say my thanks again, thnx for making this!
this is the only project that made made my intel and amd gpus to work! i mean u can even mix cpu amd intel in model section for better usage! this is the only one out there that can do, many people should know about onnxstack, spread the word whoever see this please!

till now i only used cpu mostly for other sd projects out there and they were so slower [still use for one of the models here but so much faster] but a mix of amd and intel directx12  for the other models i got in onnxstack
